tests/util/bus.test.util.js

```diff
@@ -59,5 +59,8 @@ function dropAll(callback) {
             logger.error(`could not drop Buses. Error: ${JSON.stringify(err)}`);
             callback(err);
         }
-    );
+    ).catch((error) => {
```

Comment:
I think .then(success, failure) already covers the case of .catch(error).

Reply:
Oh-- hmm.. I was dealing with un-caught promises, and this fixed it.. maybe there's a promise that doesn't have the , failure parameter?